installing/installing_gcp/installing-gcp-customizations.adoc

```diff
@@ -92,6 +92,11 @@ include::modules/cco-ccoctl-configuring.adoc[leveloffset=+3]
 //Task part 2: Creating the required GCP resources
 include::modules/cco-ccoctl-creating-at-once.adoc[leveloffset=+3]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-google-cloud-file[GCP Filestore Container Storage Interface (CSI) Driver Operator]
+
```

Comment:
This seems a bit off topic for the user reading this. You can do a lot of things with `ccoctl`, but we don't list them here - here they are just trying to install on GCP
```suggestion
```